analyzers/tests/SonarAnalyzer.UnitTest/TestCases/NativeMethodsShouldBeWrapped.CSharp9.cs

```diff
@@ -1,14 +1,31 @@
-﻿public record Record
+﻿using System.Runtime.InteropServices;
+
+public record Record
 {
     extern private static void Extern0(); // Compliant
 
     extern private static void Extern1(string s, int x); // Compliant
-    
+
     extern public static void Extern2(string s, int x); // Noncompliant {{Make this native method private and provide a wrapper.}}
-//                            ^^^^^^^
+    //                        ^^^^^^^
 
     public void Method()
     {
         extern static void Extern(string s, int x); // Compliant - local method
     }
 }
+
+namespace TestsFromDeprecatedS4214
+{
+    public record Record
+    {
+        public void Method()
+        {
+            [DllImport("kernel32.dll", CharSet = CharSet.Unicode)]
+            static extern bool RemoveDirectory(string name);     // Compliant - Method is not publicly exposed
+        }
+
+        [DllImport("kernel32.dll", CharSet = CharSet.Unicode)]
+        public static extern bool RemoveDirectory(string name);  // Noncompliant
+    }
+}
```

Comment:
Copy of https://github.com/SonarSource/sonar-dotnet/blob/c18ed76dbbc997007e41be9e9e49df3aa68ab233/analyzers/tests/SonarAnalyzer.UnitTest/TestCases/PInvokesShouldNotBeVisible.CSharp9.cs#L1-L13